packages/content-management/table_list/src/table_list_view.tsx

```diff
@@ -138,8 +162,18 @@ function TableListViewComp<T extends UserContentCommonSchema>({
         }),
         sortable: true,
         render: (field: keyof T, record: T) => (
+          // The validation is handled at the top of the component
+          // eslint-disable-next-line  @elastic/eui/href-or-on-click
```

Comment:
I'm not having any issues locally without `eslint-disable-next-line`...? Can you check and see if this can be removed? Thanks!

Reply:
That's strange, after doing `yarn kbn bootstrap` did you restart the eslint server in VS Code? (or simply restart VS Code).

<img width="764" alt="Screenshot 2022-09-07 at 08 12 33" src="https://user-images.githubusercontent.com/2854616/188812997-a0749d81-c333-43e2-ac07-fc9efebe821b.png">

Reply:
This is the eslint error that I get

<img width="928" alt="Screenshot 2022-09-07 at 08 11 14" src="https://user-images.githubusercontent.com/2854616/188813104-dc130748-0920-4080-baeb-48643a0ea9e7.png">